src/Aspire.Dashboard/Utils/BrowserStorageKeys.cs

```diff
@@ -13,6 +13,7 @@ internal static class BrowserStorageKeys
     public const string StructuredLogsPageState = "Aspire_PageState_StructuredLogs";
     public const string MetricsPageState = "Aspire_PageState_Metrics";
     public const string ConsoleLogsPageState = "Aspire_PageState_ConsoleLogs";
+    public const string ConsoleLogTimestampState = "Aspire_PageState_ConsoleLogTimestamp";
```

Comment:
It's not page state. Lets make it hold settings related to the console.
```suggestion

    public const string ConsoleLogConsoleSettings = "Aspire_ConsoleLog_ConsoleSettings";
```